### ISSUE TITLE
GraphQLResultError(path: ["idp", "authenticate", "userId"], underlying: Apollo.JSONDecodingError.couldNotConvert(value: 27, to: Swift.String))

### ISSUE THREAD

**Issue by duydtdev** [apollographql/apollo-ios#218]
I got an error when call a request to get data below
```swift 
GraphQLResultError(path: ["idp", "authenticate", "userId"], underlying: Apollo.JSONDecodingError.couldNotConvert(value: 27, to: Swift.String))
```

This is my model and query:
```swift 
query Authenticate ($username: String!, $method: String!, $credentials: [String!]!) {
  idp {
    authenticate(username: $username, method: $method, credentials: $credentials) {
      userId
      organizations { id name }
      featureLevel
      token
    }
  }
}
```

```swift
type Authenticate {
    token: String
    userId: Long
    role: Role!
    organizations: [OrganizationInfo!]
    featureLevel: Int
    data: [ CustomerCredentials ]
}

type CustomerCredentials {
    organization: String!
    token: String!
}
```
Could you help tell me why? Many thanks

**Comment by martijnwalraven** [comment]
`userId` is defined as `Long`, which is not a built-in type but a custom scalar. So it is treated as `String` by default. See [this issue](https://github.com/apollographql/apollo-ios/issues/23) for a discussion and solution.

**Comment by designatednerd** [comment]
This is fixed by #402 and will ship with `0.14.0`

**Comment by designatednerd** [comment]
Shipped with `0.14.0`!
